packages/streamline/src/components/cards/card-media/card-media.tsx

```diff
@@ -0,0 +1,38 @@
+import { ImageBackground, StyleSheet } from 'react-native';
+
+import { Box } from '../../../primitives/box/box';
+import { Card } from '../../../primitives/card/card';
+import { Text } from '../../../primitives/text/text';
+import { CardMediaProps } from './card-media.types';
+import { VideoOverlay } from './component/video-overlay';
+
+const CARD_HEIGHT = 192;
+
+export const CardMedia = ({
+  type,
+  duration,
+  onPress,
+  thumbnail,
+  caption,
+}: CardMediaProps) => {
+  return (
+    <Box>
+      <Card width="100%" height={CARD_HEIGHT} overflow="hidden">
+        <ImageBackground
```

Comment:
Won't we want to use react-native-fast-image instead of react-native Image?

Reply:
lets create a ticket for updating all our image component on streamline since we do not use `react-native-fast-image` for any of them. Alright ?

Reply:
https://www.notion.so/luko/Update-streamline-image-component-to-use-react-native-fast-image-2454b0a37d9943c9b9bcd21732d74285?pvs=4

Reply:
Alright, I let you create the ticket!